guide/src/macros/impl.md

```diff
@@ -18,9 +19,8 @@ Class methods can take a `&self` or `&mut self` parameter. They cannot take a
 consuming `self` parameter. Static methods can omit this `self` parameter.
 
 To access the underlying Zend object, you can take a reference to a
-`ZendClassObject<T>` in place of the self parameter, where the parameter is
-annotated with the `#[this]` attribute. This can also be used to return a
-reference to `$this`.
+`ZendClassObject<T>` in place of the self parameter, where the parameter must
+be named `self_`. This can also be used to return a reference to `$this`.
```

Comment:
Why not requiring `this` as a name?
Makes it both idiomatic and obvious to me.

Reply:
My future self is still sad about using `self_` instead of the idiomatic `this`...